suite-native/app/src/navigation/RootStackNavigator.tsx

```diff
@@ -25,6 +25,7 @@ import { SendStackNavigator } from '@suite-native/module-send';
 import { CoinEnablingInitScreen } from '@suite-native/coin-enabling';
 import { ConnectPopupScreen, useConnectPopupNavigation } from '@suite-native/module-connect-popup';
 import { StakingDetailScreen } from '@suite-native/module-staking-management';
+import { SettingsScreenStackNavigator } from '@suite-native/module-settings';
```

Comment:
I'm not super happy about the `SettingsScreen*` naming pattern but couldn't figure out a better one. Already tried `SettingsSection*` but that conflicts with the `SettingsSection` component which has bit different semantics. @shenkys uses _settings subpage_ but we don't have that concept in code at all (expected for pagination). Ideas? 🤔